Src/Private/Get-AbrFgtRoute.ps1

```diff
@@ -0,0 +1,169 @@
+
+function Get-AbrFgtRoute {
+    <#
+    .SYNOPSIS
+        Used by As Built Report to returns Route settings.
+    .DESCRIPTION
+        Documents the configuration of Fortinet Fortigate in Word/HTML/Text formats using PScribo.
+    .NOTES
+        Version:        0.1.0
+        Author:         Alexis La Goutte
+        Twitter:        @alagoutte
+        Github:         alagoutte
+        Credits:        Iain Brighton (@iainbrighton) - PScribo module
+
+    .LINK
+        https://github.com/AsBuiltReport/AsBuiltReport.Fortinet.Fortigate
+    #>
+    [CmdletBinding()]
+    param (
+
+    )
+
+    begin {
+        Write-PscriboMessage "Discovering Route settings information from $System."
+    }
+
+    process {
+
```

Comment:
Add If statement with `If ($InfoLevel.Route.PSObject.Properties.Value -ne 0)`. This will then skip the Route section if all the sub-sections are disabled.